src/bundle/Resources/public/js/scripts/admin.notifications.modal.js

```diff
@@ -179,9 +179,11 @@
 
     const notificationsTable = modal.querySelector(SELECTOR_TABLE);
     currentPageLink = notificationsTable.dataset.notifications;
+    let interval = Number.parseInt(notificationsTable.dataset.notificationsCountInterval);
+    if (!interval) { interval = INTERVAL; }
```

Comment:
```suggestion
    const interval = Number.parseInt(notificationsTable.dataset.notificationsCountInterval) || INTERVAL;
```

Reply:
I like this, just one small thing
`const interval = parseInt(notificationsTable.dataset.notificationsCountInterval, 10) || INTERVAL;`

Reply:
Thank you @dew326 for suggesting to add the radix/base.